build.gradle

```diff
@@ -79,7 +79,8 @@ test {
 intellij {
     pluginName "Request mapper"
     type 'IC'
-    version '2019.1'
+    version = '2020.1.1'
+    plugins = ["java", "org.jetbrains.kotlin"]
```

Comment:
What's the difference with `"org.jetbrains.kotlin:$kotlin_version-release-IJ2020.1-1"` ?

Reply:
I think they are equivalent. because Idea shows me that I use 1.3.72-release-IJ2020.1-1 org.jetbrains.kotlin